### ISSUE TITLE
Feature Request: Support for icons aka entity_picture

### ISSUE THREAD

**Issue by SeanPM5** [iantrich/radial-menu#3]
Just thought of a cool idea, to have a radial-menu that shows printer ink levels. But it doesn't seem like icons aka entity_picture appear. There's just a blank space where they are supposed to be. 

```
homeassistant:
  customize:
    sensor.epson_ink_level_yellow:
      entity_picture: /local/printer/ink_yellow.png
```
Here is a mockup of what I was attempting (pretend the mdi:remote icon in the middle is mdi:printer instead). 

![image](https://i.imgur.com/Qb3UsAN.jpg)

(My mockup displays the ink percentage levels as well, that is also not currently possible either I think, so maybe a `show_state: true` option as well?)

So basically something like this (since entity_picture would be already defined in `customize.yaml` like above example):

```
  - type: 'custom:radial-menu'
    icon: 'mdi:printer'
    name: 'Printer'
    show_state: true
    items:
      - entity: sensor.epson_ink_level_yellow
      - entity: sensor.epson_ink_level_magenta
      - entity: sensor.epson_ink_level_cyan
      - entity: sensor.epson_ink_level_black
```

This could also be useful for channel changes scripts like in this other mockup:

![image](https://i.imgur.com/3hj1kqG.jpg)

**Comment by iantrich** [comment]
I knew you would be coming with ideas 😜

**Comment by vMeph** [comment]
hell  yeahh that looks awesome @SeanPM5  lol, 
more work for you @iantrich.....

**Comment by iantrich** [comment]
@SeanPM5 you need to do a PR with cool example gifs after I do this. That's your price ;)

**Comment by SeanPM5** [comment]
@iantrich haha sounds good to me 😛

**Comment by iantrich** [comment]
Closed in v1.2.0
